packages/services/f-http/src/tests/errorHandler.test.js

```diff
@@ -0,0 +1,38 @@
+import handleError from '../errorHandler';
+
+describe('errorHandler', () => {
+    it('errorHandler should be defined', () => {
```

Comment:
No need to mention `errorHandler` in every test name since their `describe` block already mentions it.